src/github/issue-comment.ts

```diff
@@ -2,14 +2,21 @@ import { emitWebhookProcessedMetrics } from "utils/webhook-utils";
 import { CustomContext } from "middleware/github-webhook-middleware";
 import { GitHubInstallationClient } from "./client/github-installation-client";
 import { getCloudInstallationId } from "./client/installation-id";
+import { GitHubIssue, GitHubIssueCommentData } from '../interfaces/github';
```

Comment:
Did you enable editorconfig/eslint in your intellij? This should be done automatically